core/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -127,6 +128,17 @@
     public static final Setting<Integer> MAX_SLICES_PER_SCROLL = Setting.intSetting("index.max_slices_per_scroll",
         1024, 1, Property.Dynamic, Property.IndexScope);
 
+    /**
+     * The number of shard copies required to be active before proceed with the
+     * write operation or returning from an index creation operation.
```

Comment:
this only applies to index creation operation, not to write operations (especially with the name "index." in the setting...